paymentsheet/src/test/java/com/stripe/android/paymentelement/confirmation/bacs/BacsConfirmationDefinitionTest.kt

```diff
@@ -227,6 +228,7 @@ class BacsConfirmationDefinitionTest {
         assertThat(launchAction.receivesResultInProcess).isTrue()
     }
 
+    @OptIn(ExperimentalEmbeddedPaymentElementApi::class)
```

Comment:
This shouldn't be needed, otherwise merchants will need to do the same.

Reply:
It's needed because the method is calling `copy` on Appearance. We can't override copy so it will always use the primary constructor. To allow usage of copy we'd need to make Embedded a field and then manually parcelize it.

Reply:
I don't think leaving as is will be an option. This will break existing merchants. We will need to find another way.

Reply:
I think the best option would be to not add the annotation to the embedded appearance API, and only add it to the builder.

Reply:
Updated 👍